mappings/net/minecraft/block/Blocks.mapping

```diff
@@ -51,8 +51,11 @@ CLASS net/minecraft/class_1752 net/minecraft/block/Blocks
 	FIELD field_7302 SNOW_LAYER Lnet/minecraft/class_197;
 	FIELD field_7303 ICE Lnet/minecraft/class_197;
 	FIELD field_7304 SNOW Lnet/minecraft/class_197;
+	FIELD field_7305 CACTUS_BLOCK Lnet/minecraft/class_197;
 	FIELD field_7306 CLAY Lnet/minecraft/class_197;
+	FIELD field_7307 SUGARCANE_BLOCK Lnet/minecraft/class_197;
 	FIELD field_7308 JUKEBOX Lnet/minecraft/class_197;
+	FIELD field_7309 PUMPKIN_BLOCK Lnet/minecraft/class_197;
```

Comment:
this doesn’t sound right… there’s a field named PUMPKIN too

Reply:
Probably a bad match between intermediary. There are a few block fields affected by it.

Reply:
I fixed it in cfd2745 🤷